apps/fleet_graphql/lib/fleet_graphql/schema/cab.ex

```diff
@@ -0,0 +1,83 @@
+defmodule FleetGraphql.Schema.Cab do
+  use Absinthe.Schema.Notation
+  use Absinthe.Relay.Schema.Notation, :modern
+
+  node(object(:cab)) do
+    field(:type, non_null(:string))
+    field(:inserted_at, non_null(:datetime))
+    field(:updated_at, non_null(:datetime))
+  end
+
+  object(:cab_queries) do
+    field(:cabs, list_of(non_null(:cab))) do
+      resolve(&FleetGraphql.Schema.Cab.list/3)
+    end
+  end
+
+
+  object(:cab_mutations) do
+    payload(field(:add_cab)) do
+      input do
+        field(:vehicle_id, non_null(:id))
+        field(:type, :string)
+      end
+
+      output do
+        field(:result, :boolean)
+      end
+
+      resolve(&FleetGraphql.Schema.Cab.add_cab/3)
+    end
+  end
+
+##Resolvers
+
+#  def fetch(parent, %{type: :cab, id: vehicle_id}, _info) when map_size(parent) === 0 do
+#    require Ecto.Query
+#    query = Ecto.Query.from(v in Fleet.Data.Cab, where: v.id == ^vehicle_id)
+#
+#     case Fleet.Repo.one(query) do
+#      cab = %Fleet.Data.Cab{} ->
+#        {:ok, cab}
+#        nil ->
+#      {:ok, nil}
+#      end
+#  end
+
+  def list(parent, _args, _info) when map_size(parent) === 0 do
+    require Ecto.Query
```

Comment:
@muhashoa you can place `require Ecto.Query` at the top of the file, inside the module after `using ...`, may be at line 4. This will save you the need to add it in those internal code blocks